From 3f9ecd8bc1557c7e34c28ebe9fc91941036ae87d Mon Sep 17 00:00:00 2001
From: James Chapman <jachapma@redhat.com>
Date: Sun, 6 Apr 2025 21:10:46 +0000
Subject: [PATCH] Issue 6720 - Remove BDB attribute from MDB DB Monitor (#6721)

Bug description: Reference to a BDB attribute exists in DatabaseMonitorMDB
class.

Fix description: Remove it

Fixes: https://github.com/389ds/389-ds-base/issues/6720

Relates: https://github.com/389ds/389-ds-base/issues/6614

Reviewed by: @progier389 (Thank you)
---
 .../389-console/src/lib/monitor/dbMonitor.jsx  | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/src/cockpit/389-console/src/lib/monitor/dbMonitor.jsx b/src/cockpit/389-console/src/lib/monitor/dbMonitor.jsx
index 08aa1aaea..bb9950ccd 100644
--- a/src/cockpit/389-console/src/lib/monitor/dbMonitor.jsx
+++ b/src/cockpit/389-console/src/lib/monitor/dbMonitor.jsx
@@ -605,12 +605,6 @@ export class DatabaseMonitorMDB extends React.Component {
                         count = 1;
                     }
 
-                    // Build up the DB Cache chart data
-                    const dbratio = config.attrs.dbcachehitratio[0];
-                    const chart_data = this.state.dbCacheList;
-                    chart_data.shift();
-                    chart_data.push({ name: _("Cache Hit Ratio"), x: count.toString(), y: parseInt(dbratio) });
-
                     // Build up the NDN Cache chart data
                     const ndnratio = config.attrs.normalizeddncachehitratio[0];
                     const ndn_chart_data = this.state.ndnCacheList;
@@ -628,7 +622,6 @@ export class DatabaseMonitorMDB extends React.Component {
                     this.setState({
                         data: config.attrs,
                         loading: false,
-                        dbCacheList: chart_data,
                         ndnCacheList: ndn_chart_data,
                         ndnCacheUtilList: ndn_util_chart_data,
                         count,
@@ -651,10 +644,8 @@ export class DatabaseMonitorMDB extends React.Component {
     }
 
     render() {
-        let chartColor = ChartThemeColor.green;
         let ndnChartColor = ChartThemeColor.green;
         let ndnUtilColor = ChartThemeColor.green;
-        let dbcachehit = 0;
         let ndncachehit = 0;
         let ndncachemax = 0;
         let ndncachecurr = 0;
@@ -671,7 +662,6 @@ export class DatabaseMonitorMDB extends React.Component {
         );
 
         if (!this.state.loading) {
-            dbcachehit = parseInt(this.state.data.dbcachehitratio[0]);
             ndncachehit = parseInt(this.state.data.normalizeddncachehitratio[0]);
             ndncachemax = parseInt(this.state.data.maxnormalizeddncachesize[0]);
             ndncachecurr = parseInt(this.state.data.currentnormalizeddncachesize[0]);
@@ -681,14 +671,6 @@ export class DatabaseMonitorMDB extends React.Component {
                 utilratio = 1;
             }
 
-            // Database cache
-            if (dbcachehit > 89) {
-                chartColor = ChartThemeColor.green;
-            } else if (dbcachehit > 74) {
-                chartColor = ChartThemeColor.orange;
-            } else {
-                chartColor = ChartThemeColor.purple;
-            }
             // NDN cache ratio
             if (ndncachehit > 89) {
                 ndnChartColor = ChartThemeColor.green;
-- 
2.49.0

